api/package-lock.json

```diff
@@ -13,33 +13,18 @@
         "express": "^4.17.3",
         "jest": "^29.5.0",
         "jsonwebtoken": "^9.0.0",
-        "pg": "^8.9.0"
+        "pg": "^8.9.0",
+        "socket.io": "^4.6.2"
       },
       "devDependencies": {
         "nodemon": "^2.0.15",
         "supertest": "^6.3.3",
         "vitest": "^0.29.2"
       }
     },
-    "..": {
-      "name": "full-stack-react",
-      "version": "1.0.0",
-      "extraneous": true,
-      "dependencies": {
-        "bootstrap": "^5.2.3",
-        "express": "^4.18.2",
-        "jsonwebtoken": "^9.0.0",
```

Comment:
Are we still using JSON Web Tokens for Auth or no?

Reply:
Yes, we are. Were these dependencies removed from the api folder? We'll need to keep jsonwebtoken in there on the api side.

Reply:
@joegor85 got it. Okay then @jordan-carlson99 lets keep the JWT here, but otherwise  ready to go for everything else.